data/version.go

```diff
@@ -0,0 +1,139 @@
+package data
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+)
+
+type PackageJson struct {
+	Engines struct {
+		Node string `json:"node"`
+	} `json:"engines"`
+}
+
+type result struct {
+	lang    string
+	version string
+}
+
+// regex to find go version from go.mod file
+const GoVersionRegex = `go \d+(\.\d+)+`
```

Comment:
can we handle go and node in separate packages? kinda like how ventress has seperate handlers for each integration

Reply:
mainly worried about how big this file will get in the future

Reply:
oh yeah, probably a good idea.